cmd/gdeploy/commands/sync/command.go

```diff
@@ -17,39 +10,15 @@ var SyncCommand = cli.Command{
 	Action: func(c *cli.Context) error {
 		ctx := c.Context
 
-		// Ensure aws account session is valid
-		deploy.MustGetCurrentAccountID(ctx, deploy.WithWarnExpiryIfWithinDuration(time.Minute))
-
-		var cfg config.SyncConfig
-		_ = godotenv.Load()
-
-		err := envconfig.Process(ctx, &cfg)
-		if err != nil {
-			return err
-		}
-
-		ic, err := deploy.UnmarshalIdentity(cfg.IdentitySettings)
-		if err != nil {
-			panic(err)
-		}
-
-		//set up the sync handler
-		syncer, err := identitysync.NewIdentitySyncer(ctx, identitysync.SyncOpts{
-			TableName:      cfg.TableName,
-			IdpType:        cfg.IdpProvider,
-			UserPoolId:     cfg.UserPoolId,
-			IdentityConfig: ic,
-		})
-
-		if err != nil {
-			return err
-		}
-
-		clio.Info("Starting sync")
-		err = syncer.Sync(ctx)
+		_, err := deploy.ConfigFromContext(ctx)
 		if err != nil {
 			return err
 		}
+		// _, err := dc.LoadOutput(ctx)
```

Comment:
This command is awaiting reimplementation.
The previous version would have only work for developers with environment variables.

Expecting this will be implemented tomorrow